sample-pong/src/commonMain/kotlin/main.kt

```diff
@@ -0,0 +1,24 @@
+import com.soywiz.korge.Korge
+import com.soywiz.korge.scene.Module
+import com.soywiz.korge.scene.Scene
+import com.soywiz.korinject.AsyncInjector
+import com.soywiz.korma.geom.SizeInt
+import kotlin.reflect.KClass
+
+suspend fun main() = Korge(Korge.Config(module = MyModule))
+
+object MyModule : Module() {
+	// define the opening scene
+	override val mainScene: KClass<out Scene> = MenuScene::class
```

Comment:
You can omit specifying type `: KClass<out Scene>` here. It will be easier to read.